src/main/java/willy/task/TaskList.java

```diff
@@ -0,0 +1,92 @@
+package willy.task;
+
+import willy.store.TaskStore;
+import willy.task.Task;
+import willy.ui.Willy;
+
+import java.util.ArrayList;
+
+/**
+ * Handles the different actions that can be done by the bot.
+ * Bot can add tasks, remove tasks, read list of tasks,
+ * update tasks that are done and find tasks.
+ */
+public class TaskList {
+    private ArrayList<Task> listOfTasks;
```

Comment:
Perhaps this kind of multivalued variable can be named as "tasks" instead of "list of tasks" I thought?

Reply:
Ahh do you know if it is the standard way of naming lists? I named it 'listOfTasks' to make it clear  that it is a list every time I see the variable being used.